### PR TITLE
updated commands to use system variables

### PR DESCRIPTION
**Pull request by Tontonjo**

### CODE REVIEW

File: general/administration/hardware-acceleration.md [hunk 1]
```diff
@@ -151,10 +151,11 @@ Install linux-headers and run the following command.
 apt-get install linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
 ```
 
-Alternatively, run this command if you are on stretch for compatibility.
+Alternatively, run thoses commands if you are on stretch for compatibility.
 
 ```sh
-apt-get install -t stretch-backports linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
+distribution=$(. /etc/*-release;echo $VERSION_CODENAME)
+apt-get install -t $distribution-backports linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md:158]: Are these commands required on more than stretch?

Comment (Tontonjo) [reply general/administration/hardware-acceleration.md:158]: If i'm not mistaken, i had to use them on buster

File: general/administration/hardware-acceleration.md [hunk 2]
```diff
@@ -151,10 +151,11 @@ Install linux-headers and run the following command.
 apt-get install linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
 ```
 
-Alternatively, run this command if you are on stretch for compatibility.
+Alternatively, run thoses commands for compatibility.
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md:154]:
```suggestion
Alternatively, run the following commands for compatibility.
```

What do they mean by compatibility? Maybe we can use a better word

Comment (Tontonjo) [reply general/administration/hardware-acceleration.md]: I did wrote the original PR for this but this wasn't in it.
I used the formulation "Alternatively, use backports"

**Pull request merged**